api/v1/recipe/views.py

```diff
@@ -133,3 +133,28 @@ def post(self, request, *args, **kwargs):
             except:
                 return Response({'error': '2', 'response': 'Bad URL or URL not supported'})
         return Response({'error': '3', 'response': 'No URL given.'})
+
+class RatingViewSet(viewsets.ModelViewSet):
```

Comment:
This model class should be read only. The users shouldn't be making edits or creating recipes from this endpoint. We can also remove the `permission_classes` once this is done.

http://www.django-rest-framework.org/api-guide/viewsets/#readonlymodelviewset